distribution/lib/Standard/Base/0.0.0-dev/src/Meta/Enso_Project.enso

```diff
@@ -52,33 +82,15 @@ type Project_Description
     namespace : Text
     namespace self = self.prim_config.namespace
 
-## Returns the Enso project description for the given module. If no module is
-   given, returns the description of the project that the engine was executed
-   with, i.e., the project that contains the `main` method, or throws
-   `Module_Not_In_Package_Error` if there is no such project, e.g., when
-   executing a single file.
-
-   Arguments:
-   - module: An optional module for which the Enso project description
-     will be fetched. If `Nothing`, the current project description will
-     be fetched.
+## Returns the Enso project description for the project that the engine was
+   executed with, i.e., the project that contains the `main` method, or
+   throws `Module_Not_In_Package_Error` if executing a single file.
 
    > Example
      Get the project description for the project that contains the `main`
      method.
 
          enso_project
-
-   > Example
-     Get the project description for the `Standard.Visualizations` project.
-
-         enso_project Standard.Visualizations
-
-   > Example
-     Get the project description for the `Standard.Base` project from the
-     `Vector` module.
-
-         enso_project Standard.Base.Data.Vector
-enso_project : (Any | Nothing) -> Project_Description ! Module_Not_In_Package_Error
-enso_project module=Nothing =
-    Project_Description.enso_project_builtin module
+enso_project : Project_Description ! Module_Not_In_Package_Error
+enso_project =
+    Project_Description.enso_project_builtin Nothing
```

Comment:
```suggestion
    Project_Description.new Nothing
```

In case any further logic is added to `new`.

Reply:
Has to be there as it moves one frame up to find where it was called from. 

If we pass through new it then ends up saying where Standard.Base is.